src/tests/test_index_btree.cpp

```diff
@@ -494,31 +304,37 @@ TYPED_TEST(BtreeTest, CpFlush) {
     const auto num_entries = SISL_OPTIONS["num_entries"].as< uint32_t >();
     LOGINFO("Do Forward sequential insert for {} entries", num_entries / 2);
     for (uint32_t i = 0; i < num_entries; ++i) {
-        this->put(i, btree_put_type::INSERT_ONLY_IF_NOT_EXISTS);
+        this->put(i, btree_put_type::INSERT);
```

Comment:
suggestion: would you please add randomly `test_common::HSTestHelper::trigger_cp(true /* wait */);` inside the loop. for example:

>  static thread_local std::uniform_int_distribution< uint32_t > random_flush_time{1, 100};
   for (uint32_t i{start_index}; i < num_entries; ++i) {
       this->put(i, btree_put_type::INSERT_ONLY_IF_NOT_EXISTS);
       if (random_flush_time(g_re)> 30){
            test_common::HSTestHelper::trigger_cp(true /* wait */);
       }
   }

Reply:
we already have a timer based trigger cp in cp mgr